bin/iotAgentTester.js

```diff
@@ -261,9 +267,11 @@ function provisionDevice(commands) {
         request(options, function(error, result, body) {
             if (error) {
                 callback('Couldn\'t connect with the provisioning server: ' + error.toString());
-            } else if (result.statusCode === 200 && body) {
+            } else if ( (result.statusCode === 200 || result.statusCode === 201) && body) {
```

Comment:
It seems there is a blank space just before the first group of conditions you may want to get rid of... ;) I mean changing `} else if ( (result.statusCode === 200 || result.statusCode === 201) && body) {` into `} else if ((result.statusCode === 200 || result.statusCode === 201) && body) {` :)